main.cc

```diff
@@ -524,6 +524,26 @@ sharded<locator::snitch_ptr> *the_snitch;
 sharded<service::storage_proxy> *the_storage_proxy;
 }
 
+static void initialize_local_info(sharded<db::system_keyspace>& sys_ks,
```

Comment:
Please, write a patch comment describing what's the point of this change. Are you going to re-use this function in one of the next patches?

Reply:
>> Are you going to re-use this function in one of the next patches?

No, the giant `scylla_main` just makes me scared and I can't reason effectively about the code in that form. So it's just a readability refactoring.